README.md

```diff
@@ -35,27 +35,7 @@ const config = {
 
 <details><summary markdown="span"><strong>For LoopBack 3 users</strong></summary>
```

Comment:
Now that the content is just a single sentence, I feel we no longer need to hide it via `<details>` tag. Let's remove this line and perhaps make the next sentence in bold?